ReactAndroid/src/main/java/com/facebook/react/devsupport/RedBoxDialog.java

```diff
@@ -323,7 +323,7 @@ public View getContentView() {
   }
 
   public boolean isShowing() {
-    return mDialog.isShowing();
+    return mDialog != null && mDialog.isShowing();
```

Comment:
RNTester was crashing without this extra check
![image](https://user-images.githubusercontent.com/11707729/153111726-cb19c10b-278c-4ad3-9bd8-e9ef0318b456.png)